packages/on-chain/contracts/interfaces/RealitioV3.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
should we mention that this file was imported from other repo.
maybe something like this
```
File imported/taken directly from ${URL} as it is with no modification.
```
or maybe a better way to put it.

Might help in reviewing as well as reference for others.